ui/widgets.go

```diff
@@ -35,6 +37,11 @@ type inputs struct {
 
 type combined struct {
 	sel *decredmaterial.Select
+
+	transactions []struct {
+		data    interface{}
+		gesture *gesture.Click
+	}
```

Comment:
Use a plain slice of icons, you can just use a green + and red - for now. We'll add the proper icons later